crf/src/test/java/org/sagebase/crf/step/active/HeartRateSampleProcessorTest.kt

```diff
@@ -301,11 +262,13 @@ class HeartRateSampleProcessorTest {
         // Only check some of the results b/c the calculations are for a resting heart rate and all use
         // initial estimate or else they are invalid (Initial windows).
         for (ii in 0 until Math.min(10, expectedOutputs.size)) {
-            val (hr, confidence) = processor.calculateHRFromFilteredSamples(inputChunks[ii], samplingRate)
+            //let filteredData = processor.getFilteredSignal(inputChunks[ii], samplingRate: roundedSamplingRate)
```

Comment:
remove or TODO:

Reply:
Again, I want to leave in the "hints" about porting for now.